### ISSUE TITLE
Transfer orbit

### ISSUE THREAD

**Issue by joelgeorgem** [jp-praveen/Rendezvous#12]
I checked the orbital parameters of the transfer orbit generated by the book method and the Prussing method (as you did).

Noticed that the book method gave a consistent transfer trajectory while the Prussing method did not. Check the implementation of the method.

By consistent trajectory, I mean the following: when I retrieve the orbital parameters providing (r1,v1) and (r2,v2), I get the same values.

**Comment by jp-praveen** [comment]
Dear Sir, 
    Finding the semi-major axis in Prussing method is just a root solving problem. I implemented the Prussing method using secant method and fzero. Both gave very high values of semi-major axes in the order of 10^20. I think the problem is either with the function or with the methods used. The fzero function when it stopped gave the function value to be around 180 (which should have been 0).

**Comment by joelgeorgem** [comment]
What I noticed is that the numerical solution of this equation is sensitive to the initial condition.

If you use as the initial guess a0, the value of 'a' obtained for the transfer orbit from the book method, then the iteration converges.

**Comment by jp-praveen** [comment]
Yes sir. I tried plotting the function, by varying 'a' for a fixed dt and found the approximate zero of the function manually. The function takes positive and negative values, i.e has a zero, only for some values of dt. I then used such dt as input transfer time and gave initial a0 near the zero which I  got from the graph. But still I got different answers from both methods, but they both converged. I will once again write the code and try again, maybe it will work.
